### PR TITLE
Added the controller file creation

### PR DESCRIPTION
**Pull request by christybowen**
This can now create controller files for a given model. Also updated Npgsql to the latest version.

### CODE REVIEW

File: PostgresSchemaGenerator/Program.cs [hunk 1]
```diff
@@ -17,7 +17,7 @@ class Program
 
         static void Main(string[] args)
         {
-            using (var conn = new NpgsqlConnection("host=sand6;Username=cbowen;Database=payledger"))
+            using (var conn = new NpgsqlConnection("host=sand5;Username=cbowen;Database=payledger"))
```
Comment (JasonSHD) [inline PostgresSchemaGenerator/Program.cs]: Why did we change back to sand5?

File: PostgresSchemaGenerator/src/Library/SchemaInterpreter.cs [hunk 2]
```diff
@@ -89,25 +91,238 @@ public void createModelString()
...
+            controllerPrintString += "                conn.Open();\n\n";
+
+            controllerPrintString += "                if (conn.State == ConnectionState.Closed)\n";
+            controllerPrintString += "                {\n";
+            controllerPrintString += "                    return StatusCode(HttpStatusCode.InternalServerError);\n";
+            controllerPrintString += "                }\n\n";
+
+            controllerPrintString += "                using (var cmd = new NpgsqlCommand())\n";
+            controllerPrintString += "                {\n";
+            controllerPrintString += "                    cmd.Connection = conn;\n";
```
Comment (JasonSHD) [inline PostgresSchemaGenerator/src/Library/SchemaInterpreter.cs]: I absolutely love this... You've built in the correct indenting on the generated class...

File: PostgresSchemaGenerator/Program.cs (hunk 1 above)
Comment (christybowen) [inline PostgresSchemaGenerator/Program.cs]: sand6 doesn't have the payledger database, which is what we have been connecting to

File: PostgresSchemaGenerator/src/Library/SchemaInterpreter.cs (hunk 2 above)
Comment (christybowen) [inline PostgresSchemaGenerator/src/Library/SchemaInterpreter.cs]: It would drive me crazy if I didn't!

**Pull request merged**